.claude/hooks/bash-worktree-fix.sh

```
@@ -0,0 +1,182 @@
+    if [ -z "${cmd##*[! 	]*}" ]; then
+        # Note: literal space and tab between [! 	]
+        debug_log "Skipping: empty/whitespace-only command"
+        return 0
+    fi
+
+    _oldset="$-"
+    set -f
+
+    # Starts with cd?
+    case "${cmd}" in
+        [ 	]cd|cd|[ 	]cd[ 	]*|cd[ 	]*)
```

Shellcheck (0.10.0)

[error] Couldn't parse this case item. Fix to allow more checks.

(SC1073)